srcs/layouts/latn_azerty_be.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Adapted from Belgian AZERTY -->
+<keyboard name="AZERTY (Belgian)" script="latin">
+  <row>
+    <key key0="a" key2="1" key4="loc esc"/>
+    <key key0="z" key1="|" key2="2" key3="&amp;" key4="\@"/>
+    <key key0="e" key2="3" key3="&quot;" key4="\#"/>
+    <key key0="r" key2="4" key3="'"/>
+    <key key0="t" key2="5" key3="(" key4=")"/>
+    <key key0="y" key2="6" key3="§" key4="^"/>
+    <key key0="u" key2="7" key3="{" key4="}"/>
+    <key key0="i" key2="8" key3="!" key4="ç"/>
+    <key key0="o" key1="loc accent_ogonek" key2="9" key3="à" key4="loc accent_cedille"/>
+    <key key0="p" key2="0" key3="-" key1="_"/>
+  </row>
+  <row>
```

Comment:
`~` is missing on the keyboard. It's useful to type with the compose key and might be used in passwords.
`accent_tilde` is not a substitute because it doesn't work with the compose key.